Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:15.04
+
+ENV RUNTIME_PACKAGES="python3 postgresql python-psycopg2 git"
+ENV BUILD_PACKAGES="curl build-essential python3-dev libpq-dev ca-certificates libffi-dev libssl-dev"
+
+WORKDIR /code
+
+EXPOSE 5000
```

Comment:
Yeh, it makes it clear what ports the container can expose even if your not exposing them